package.json

```
@@ -9,13 +9,21 @@
   },
   "license": "ISC",
   "bin": {
-    "create-kadena-app": "lib/index.js"
+    "create-kadena-app": "./bin/index.js"
   },
   "files": [
     "dist",
     "lib"
   ],
+  "contributors": [{
+    "name": "Jermaine de Jong",
```

```suggestion
    "name": "Jermaine Jong",
```
Thanks for adding, small typo fix :-)